src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -294,6 +295,91 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
     }
   }
 
+  test("Verify quick optimize rebuild of index after index incremental refresh.") {
+    withTempPathAsString { testPath =>
+      val indexConfig = IndexConfig(s"index", Seq("RGUID"), Seq("imprs"))
+      import spark.implicits._
+      SampleData.testData
+        .toDF("Date", "RGUID", "Query", "imprs", "clicks")
+        .limit(10)
+        .write
+        .parquet(testPath)
+      val df = spark.read.parquet(testPath)
+      hyperspace.createIndex(df, indexConfig)
+      var indexCount =
+        spark.read
+          .parquet(s"$systemPath/index" +
+            s"/${IndexConstants.INDEX_VERSION_DIRECTORY_PREFIX}=0")
+          .count()
+      assert(indexCount == 10)
+
+      // Change original data.
+      SampleData.testData
+        .toDF("Date", "RGUID", "Query", "imprs", "clicks")
+        .limit(3)
+        .write
+        .mode("append")
+        .parquet(testPath)
+      hyperspace.refreshIndex(indexConfig.indexName, REFRESH_MODE_INCREMENTAL)
+      val newIndexLocation = s"$systemPath/index"
```

Comment:
let's remove this variable since it's misleading (the new location is `=1`).

Reply:
done